Makefile

```diff
@@ -8,20 +8,25 @@
 
 SIL ?= @
 MAKEFLAGS += --no-print-directory
+VERSION="1.21 rev: "`git rev-parse --short HEAD`
```

Comment:
Don't use backticks in Makefile, use proper `$(shell something)`

Rigth now works by accident (it's being copied to CFLAGS and `git rev-parse` is being run with each single gcc run)

Reply:
Fixed